app/controllers/api/users_controller.rb

```diff
@@ -3,7 +3,8 @@ class UsersController < Api::ApplicationController
     skip_before_action :doorkeeper_authorize!, only: %i[create]
 
     def create
-      user = User.new(name: user_params[:name], email: user_params[:email], password: user_params[:password])
+      user = User.new(name: user_params[:name], email: user_params[:email], password: user_params[:password],
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍